staking/cli/src/main.rs

```diff
@@ -97,5 +99,14 @@ fn main() {
         Action::SaveStakeAccountsSnapshot {} => {
             save_stake_accounts_snapshot(&rpc_client);
         }
+        Action::ClaimRewards {
+            min_staked,
+            min_reward,
```

Comment:
I don't like having two parameters here. I feel like we should be disciplined and:
- either claim all rewards for accounts with min_staked
- or claim all rewards s.t. reward > min_reward